### ISSUE TITLE
Loop with one item

### ISSUE THREAD

**Issue by PixelT** [smashingboxes/OwlCarousel2#1196]
loop: true when in slider is only one item return error:
```
TypeError: d[c[(c.length - 1)]] is undefined
```

**Comment by romanlex** [comment]
confirm bug

**Comment by SergiSMJ** [comment]
Same here, I fixed the bug with this code which I think is a better solution than the stackoverflow one:

`$('.owl-carousel').owlCarousel({`
`  margin: 30,`
`  padding: 30,`
`  dots: true,`
`  center: true,`
`  loop: $(this).find('.owl-item').length > 1 true : false,`
`  items: 1`
`});`

Is strange put a carousel with one item only, but in my case I'm using the plugin in a CMS and the users are able to put onlye one item if they want.

**Comment by daviddeutsch** [comment]
Yeah, this should definitely be a feature where OWL unsets the loop value with no items. Preferably something that needs to be checked on again when items are added or removed, too.

**Comment by fe-rod** [comment]
I made a very very very basic fix for this. You can see my reply here: #1200 

I'm using the carousel with Angular 2, so I tried to avoid things like `$(this).find('.owl-item').length`. My solution, considering I'm using multiple carousels with the same behavior, was to modify the default setting of the carousel. As @daviddeutsch says the solution should consider when items are added or removed, but this solution is not the case.

The good thing though, is that it works with the responsive options, because the setup function is called when the carousel updates.
